flake.nix

```diff
@@ -96,18 +96,25 @@
 
       checks = forAllSystems (
         system:
-        {
-          tarball = jobs.${system}.tarball;
-        }
+        { }
+        //
+          lib.optionalAttrs
+            (
+              # Exclude x86_64-freebsd because "Failed to evaluate rustc-wrapper-1.85.0: «broken»: is marked as broken"
+              system != "x86_64-freebsd"
+            )
+            {
+              tarball = jobs.${system}.tarball;
+            }
         //
           lib.optionalAttrs
             (
               self.legacyPackages.${system}.stdenv.hostPlatform.isLinux
               # Exclude power64 due to "libressl is not available on the requested hostPlatform" with hostPlatform being power64
               && !self.legacyPackages.${system}.targetPlatform.isPower64
-              # Exclude armv6l-linux due to "cannot bootstrap GHC on this platform ('armv6l-linux' with libc 'defaultLibc')"
+              # Exclude armv6l-linux because "cannot bootstrap GHC on this platform ('armv6l-linux' with libc 'defaultLibc')"
               && system != "armv6l-linux"
-              # Exclude riscv64-linux due to "cannot bootstrap GHC on this platform ('riscv64-linux' with libc 'defaultLibc')"
+              # Exclude riscv64-linux because "cannot bootstrap GHC on this platform ('riscv64-linux' with libc 'defaultLibc')"
```

Comment:
cherry-pick check failed because these conditionals were already present in 24.11. IMO that's fine and expected.

(I didn't realize these 2 conditionals were already present in 24.11, and when I added the comments in the original PR, I used `because` instead of `due to` which caused the cherry-pick automation to fail lol)